docs2/site/docs/migrations/migration5.md

```diff
@@ -108,6 +129,17 @@ private class CustomAutoObjectType<T> : AutoRegisteringObjectGraphType<T>
 Similarly, by overriding `CreateField` you can change the default name, description,
 graph type, or other information applied to each generated field.
 
+These `protected` methods can be overridden to provide the following customizations to
+automatically-generated graph types:
+
+| Method           | Description                                                          | Typical use      |
+|------------------|----------------------------------------------------------------------|------------------|
+| (constructor)    | Configures graph properties and adds fields                          | Configuring graph after default initialization is complete |
+| ConfigureGraph   | Configures default graph properties prior to applying attributes     | Applying a different default naming convention, such as appending "Input" or "Model" |
```

Comment:
Maybe after applying attributes to be able to override them ? I'm not sure about priorities here.

Reply:
Constructor covers that purpose. This method specifically exists to apply defaults before the attributes get applied.

Reply:
ok